PublicSDKSwiftAPITester/AttributionNetworkAPI.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  Copyright RevenueCat Inc. All Rights Reserved.
+//
+//  Licensed under the MIT License (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      https://opensource.org/licenses/MIT
+//
+//  AttributionNetworkAPI.swift
+//
+//  Created by Madeline Beyl on 8/25/21.
+
+import Foundation
+import Purchases
+
+func checkRCAttributionNetworkEnums() {
+    var aNetwork: RCAttributionNetwork = RCAttributionNetwork.appleSearchAds
```

Comment:
🤔 
for enums, I think a more robust way to handle the API test is to use the value in a `switch` statement, and not set a default so that compilation will fail to compile if we add a new value